packages/api-derive/src/elections/approvalsOf.ts

```diff
@@ -28,10 +28,16 @@ export function approvalsOf (api: ApiInterfaceRx): (who: AccountId) => Observabl
     (api.query.elections.nextVoterSet<SetIndex>())
       .pipe(
         switchMap((nextVoterSet: SetIndex): Observable<Vec<ApprovalFlag>[]> =>
-          api.query.elections.approvalsOf.multi([...Array(nextVoterSet.toNumber() + 1).keys()].map((i): [string, number] => [who.toString(), i])) as any as Observable<Vec<ApprovalFlag>[]>
+          api.query.elections.approvalsOf.multi(
```

Comment:
Just reformatting to make it easier to follow, aka nothing-to-see-here-move-along